<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>1033</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Patient Information</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>Health</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>Remark</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>0</y>
           <width>749</width>
           <height>189</height>
          </rect>
         </property>
         <property name="title">
          <string>Control Button</string>
         </property>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>91</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>80</y>
            <width>91</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>End</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>20</y>
            <width>241</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Swallow</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>80</y>
            <width>241</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Cough</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="title">
       <string>Camera</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2" rowspan="4">
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>450</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="text">
          <string>24</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Device</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_3"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Sample Rate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 127);</string>
      </property>
      <property name="title">
       <string>Microphone</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Device</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QWidget" name="widget_4" native="true">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>190</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Sample Rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="text">
          <string>44100</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="title">
       <string>IMU</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,2,1,6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Sample Rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="text">
          <string>1000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
